bullet_train/app/helpers/account/dates_helper.rb

```diff
@@ -1,40 +1,21 @@
 module Account::DatesHelper
-  # e.g. October 11, 2018
-  def display_date(timestamp, custom_date_format = nil)
-    return nil unless timestamp
-    if custom_date_format
-      local_time(timestamp).strftime(custom_date_format)
-    elsif local_time(timestamp).year == local_time(Time.now).year
-      local_time(timestamp).strftime("%B %-d")
-    else
-      local_time(timestamp).strftime("%B %-d, %Y")
-    end
+  def display_date(timestamp, format: :defaul)
```

Comment:
Typo: **_:defaul_** should be :default?

Reply:
Just noticed this, about to push changes here in a bit